ckanext/georchestra/utils/organizations.py

```diff
@@ -98,6 +99,9 @@ def organization_set_member_or_create(context, user_id, org_id, role):
     :param role:
     :return:
     """
+    # we accept orgs that doesn't exist, but we don't accept user without any orgs.
+    if org_id == '':
+        abort(403)
```

Comment:
Looks like passing a message as second argument is possible, wouldn't it be useful?